src/XMakeTasks/GetFrameworkSDKPath.cs

```diff
@@ -38,15 +38,15 @@ public string Path
             {
                 if (s_path == null)
                 {
-                    s_path = ToolLocationHelper.GetPathToDotNetFrameworkSdk(TargetDotNetFrameworkVersion.VersionLatest, VisualStudioVersion.VersionLatest);
+                    s_path = ToolLocationHelper.GetPathToDotNetFrameworkSdk(TargetDotNetFrameworkVersion.Latest, VisualStudioVersion.VersionLatest);
 
                     if (String.IsNullOrEmpty(s_path))
                     {
                         Log.LogMessageFromResources(
                             MessageImportance.High,
                             "GetFrameworkSdkPath.CouldNotFindSDK",
-                            ToolLocationHelper.GetDotNetFrameworkSdkInstallKeyValue(TargetDotNetFrameworkVersion.VersionLatest, VisualStudioVersion.VersionLatest),
-                            ToolLocationHelper.GetDotNetFrameworkSdkRootRegistryKey(TargetDotNetFrameworkVersion.VersionLatest, VisualStudioVersion.VersionLatest)
+                            ToolLocationHelper.GetDotNetFrameworkSdkInstallKeyValue(TargetDotNetFrameworkVersion.Latest, VisualStudioVersion.VersionLatest),
```

Comment:
>GetDotNetFrameworkSdkInstallKeyValue [](start = 47, length = 36)

Update the GetDotNetFrameworkSdkInstallKeyValue tests to assert it also works with Latest